mo_net/samples/cbow.py

```
@@ -0,0 +1,426 @@
+    if not model_path.exists():
+        raise click.ClickException(f"Model file not found: {model_path}")
+
+    vocab_path = model_path.with_suffix(".vocab.npy")
+    if not vocab_path.exists():
+        raise click.ClickException(f"Vocabulary file not found: {vocab_path}")
+
+    vocab_data = np.load(vocab_path, allow_pickle=True).item()
+    vocab = vocab_data["vocab"]
+    token_to_id = vocab_data["token_to_id"]
+
+    model = Model.load(open(model_path, "rb"), training=False)
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)